tempesta_fw/http.c

```diff
@@ -834,16 +856,24 @@ tfw_http_enum_resp_code(int status)
 }
 
 static inline void
-tfw_http_error_resp_switch(TfwHttpReq *req, int status, const char *reason)
+tfw_http_error_resp_switch(TfwHttpReq *req, int status)
 {
 	resp_code_t code = tfw_http_enum_resp_code(status);
 	if (code == RESP_NUM) {
 		TFW_WARN("Unexpected response error code: [%d]\n", status);
-		tfw_http_send_resp(req, RESP_500);
+		__tfw_http_send_resp(req, RESP_500);
 		return;
 	}
+	__tfw_http_send_resp(req, code);
+}
 
-	tfw_http_send_resp(req, code);
+/* Common interface for sending error responses. */
+void
+tfw_http_send_resp(TfwHttpReq *req, int status, const char *reason)
+{
+	if (!(tfw_blk_flags & TFW_BLK_ERR_NOLOG))
+		TFW_WARN_ADDR(reason, &req->conn->peer->addr);
```

Comment:
Please add the `status` to the warning message, to get something like:
```
[tempesta] Warning: request dropped: processing error, status 502: ::ffff:c0a8:6401
```
For more verbose logging we have #537, but now we still need something more administration-friendly.

Reply:
Corrected.